Firestore/core/src/bundle/bundle_reader.cc

```diff
@@ -121,7 +121,7 @@ void BundleReader::ReadJsonToBuffer(size_t required_size) {
   while (buffer_.size() < required_size) {
     // Read at most 1024 bytes every time, to avoid allocating a huge buffer
     // when corruption leads to large `required_size`.
-    auto size = std::min(1024ul, required_size - buffer_.size());
+    auto size = std::min<unsigned long>(1024ul, required_size - buffer_.size());
```

Comment:
The linter is not happy:
```
> /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/scripts/check_lint.py 8e79c748219a4fc9969168212d3ae899ce87ac30
Firestore/core/src/bundle/bundle_reader.cc:124:  Use int16/int64/etc, rather than the C type long  [runtime/int] [4]
```
Perhaps use `size_t` as the template argument instead?

Reply:
Sure, will do.